lib/cli-engine.js

```diff
@@ -489,7 +489,10 @@ CLIEngine.getFormatter = function(format) {
         try {
             return require(formatterPath);
         } catch (ex) {
-            return null;
+            if (ex.code === "MODULE_NOT_FOUND" && ex.message.indexOf("'" + formatterPath + "'") !== -1) {
```

Comment:
That sounds like a good idea. The reason I didn't go there was that it is more of a breaking change, than just relaying some errors. I added a new commit on this branch where I make this change.